cli/src/pcluster/utils.py

```diff
@@ -36,18 +36,48 @@
 
 LOGGER = logging.getLogger(__name__)
 
+DEFAULT_PARTITION = "aws"
+PARTITION_MAP = {
+    "cn": "aws-cn",
+    "us-gov": "aws-us-gov",
```

Comment:
why not`us-gov-`?

Reply:
No specific reason. I agree to use `us-gov-` instead of `us-gov`.